lib/integrations/slack.rb

```diff
@@ -1,6 +1,51 @@
 class Integrations::Slack < Integrations::Base
   SUPPORTED_EVENTS = %w(run_completion run_error webhook_timeout run_test_failure).freeze
 
+  def message_text(fallback = false)
+    message = self.send(event_type.dup.concat("_message").to_sym)
+    if fallback
+      "Your Rainforest Run #{message}"
+    else
+      "Your Rainforest Run (<#{payload[:frontend_url]} | Run ##{run[:id]}#{run_description}>) #{message}"
+    end 
+  end
+
+  def run_description
+    run[:description].present? ? ": #{run[:description]}" : ""
```

Comment:
same thing here - you can probably leave this logic it in the method where it's called.